specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-10-01-preview/AutomationRules.json

```diff
@@ -200,293 +247,233 @@
+          "description": "The display name of the automation rule",
           "type": "string"
         },
         "order": {
+          "format": "int32",
           "description": "The order of execution of the automation rule",
-          "type": "integer",
-          "format": "int32"
+          "type": "integer"
         },
         "triggeringLogic": {
-          "$ref": "#/definitions/AutomationRuleTriggeringLogic",
-          "description": "The triggering logic of the automation rule",
-          "type": "object"
+          "$ref": "#/definitions/AutomationRuleTriggeringLogic"
         },
         "actions": {
           "description": "The actions to execute when the automation rule is triggered",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/AutomationRuleAction"
-          },
-          "type": "array"
-        },
-        "createdTimeUtc": {
-          "description": "The time the automation rule was created",
-          "format": "date-time",
-          "readOnly": true,
-          "type": "string"
+          }
         },
         "lastModifiedTimeUtc": {
-          "description": "The last time the automation rule was updated",
           "format": "date-time",
-          "readOnly": true,
-          "type": "string"
+          "description": "The last time the automation rule was updated",
+          "type": "string",
+          "readOnly": true
         },
-        "createdBy": {
-          "$ref": "../../../common/2.0/types.json#/definitions/ClientInfo",
-          "description": "Describes the client that created the automation rule",
-          "readOnly": true,
-          "type": "object"
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "The time the automation rule was created",
+          "type": "string",
+          "readOnly": true
         },
         "lastModifiedBy": {
-          "$ref": "../../../common/2.0/types.json#/definitions/ClientInfo",
-          "description": "Describes the client that last updated the automation rule",
           "readOnly": true,
-          "type": "object"
-        }
-      },
-      "required": [
-        "displayName",
-        "order",
-        "triggeringLogic",
-        "actions"
-      ],
-      "type": "object"
-    },
-    "AutomationRulesList": {
-      "description": "List all the automation rules.",
-      "properties": {
-        "nextLink": {
-          "description": "URL to fetch the next set of automation rules.",
-          "readOnly": true,
-          "type": "string"
+          "$ref": "../../../common/2.0/types.json#/definitions/ClientInfo"
         },
-        "value": {
-          "description": "Array of automation rules.",
-          "items": {
-            "$ref": "#/definitions/AutomationRule"
-          },
-          "type": "array"
+        "createdBy": {
+          "readOnly": true,
+          "$ref": "../../../common/2.0/types.json#/definitions/ClientInfo"
         }
```

Comment:
Were these added before systemData was introduced?

These metadata should be covered by systemData.

[https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html](https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html)

Reply:
Hi, This resource is not new, and was introduced in 2019-01-01 version.

Reply:
I see. SystemData should be supported in any new api-versions. It will be duplicate data in your case, but the intent is to have a common schema for system metadata across RPs.

Reply:
So, yes we have duplication in this case.
You can see that these fields are inside "AutomationRuleProperties" which is part of "AutomationRule" which has all the fields of "ResourceWithEtag" which has all the fields of Resource which contains SystemData.